pallets/committee/src/lib.rs

```diff
@@ -351,26 +362,35 @@ pub mod pallet {
 		/// This can only be called by members of the committee.
```

Comment:
```suggestion
		/// This can only be called by members of the committee that are eligible to vote.
		/// New members are eligible to vote after 1 voting period has passed from the block they were added to the members set.
```